docs/installation.md

```
@@ -0,0 +1,64 @@
+dart pub add shalom_core:'{"git":{"url": "https://github.com/nrbnlulu/shalom.git", "path": "dart/shalom_core"}}'
+dart pub add shalom_flutter:'{"git":{"url": "https://github.com/nrbnlulu/shalom.git", "path": "dart/shalom_flutter"}}'
+```
+
+If you are not using Flutter, you can omit `shalom_flutter`.
+
+## Project layout
+
+Shalom expects your project to include a GraphQL schema and operation files. A common
+layout is:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)